lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -124,8 +125,10 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
         .pipe(
             debounceTime(400),
             distinctUntilChanged(),
+            tap(() => this.isCreated = false),
```

Comment:
Why do you need to add the tap for created = true/false? to check if the form is valid?

Reply:
before calling the API, this makes the start button to  disabled.

Reply:
resolved thanks